crates/aptos/src/move_tool/mod.rs

```diff
@@ -594,6 +599,43 @@ impl CliCommand<&'static str> for ListPackage {
     }
 }
 
+/// Cleans derived artifacts of a package.
+#[derive(Parser)]
+pub struct CleanPackage {
+    #[clap(flatten)]
+    pub(crate) move_options: MovePackageDir,
+    #[clap(flatten)]
+    pub(crate) prompt_options: PromptOptions,
+}
+
+#[async_trait]
+impl CliCommand<&'static str> for CleanPackage {
+    fn command_name(&self) -> &'static str {
+        "Clean"
+    }
+    async fn execute(self) -> CliTypedResult<&'static str> {
+        let path = self.move_options.get_package_path()?;
+        let build_dir = path.join("build");
+        std::fs::remove_dir_all(build_dir)
+            .map_err(|e| CliError::IO("Removing Move build dir".to_string(), e))?;
+
+        let move_dir = &*MOVE_HOME;
+        if prompt_yes_with_override(
+            &format!(
+                "Do you also want to delete the local package download cache at `{}`?",
+                move_dir
+            ),
+            self.prompt_options,
+        )
```

Comment:
You can just put a `?` at the end of prompt_yes_with_override, and it will just say `Aborted` if you say no.

Oh, but I see you want to keep the concept of still successful if the download cache is skipped, but the build was cleaned.

Reply:
Yes, I wanted to preserve the success here. Its not like an abort as in other cases.